compiler/src/iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp

```diff
@@ -1389,11 +1389,16 @@ getPackVectorTileSizes(mlir::FunctionOpInterface entryPointFn,
   SmallVector<int64_t> tileSizes(op.getSourceRank(), 1);
   auto targetAttr = IREE::HAL::ExecutableTargetAttr::lookup(entryPointFn);
   int64_t vectorSize = getVectorSize(entryPointFn, op.getSourceType());
+  if (!(hasAVX512fFeature(targetAttr) && isPackMatmulLHS(op))) {
```

Comment:
nit: we can get rid of `!()` with `||`. E.g.,

```suggestion
  if (!hasAVX512fFeature(targetAttr) || !isPackMatmulLHS(op)) {
```